test/test_core.py

```diff
@@ -7722,6 +7722,41 @@ def test_source_map(self, args):
     else:
       self.assertTrue(seen_lines.issuperset([6, 7, 11, 12]), seen_lines)
 
+  @no_asan("asan does not work with dynamic linking")
```

Comment:
is that true?   I don't see any other dylink tests marked like this.

Reply:
The error message may be inaccurate here. I'll see if I can reproduce the test failure again.

Reply:
Looks like @no_asan is no longer required.